frontend/pages/beta/model/new.tsx

```diff
@@ -94,27 +94,29 @@ export default function NewModel() {
               <Typography component='h2' variant='h6'>
                 Overview
               </Typography>
-              <Stack direction='row' spacing={2}>
+              <Box sx={{ width: '100%' }}>
                 <TeamAndModelSelector
                   setTeamValue={setTeamName}
                   teamValue={teamName}
                   setModelValue={setModelName}
                   modelValue={modelName}
                 />
-              </Stack>
+              </Box>
               <Stack>
                 <FormControl>
-                  <Typography component='label' sx={{ fontWeight: 'bold' }} htmlFor={'new-model-description'}>
-                    Description <span style={{ color: theme.palette.primary.main }}>*</span>
-                  </Typography>
-                  <TextField
-                    id='new-model-description'
-                    required
-                    size='small'
-                    value={description}
-                    onChange={(e) => setDescription(e.target.value)}
-                    data-test='modelDescription'
-                  />
+                  <Box id='new-model-description'>
```

Comment:
If you need an ID you should be able to add it to the RichTextEditor component below and remove this Box